translations/fr.json

```diff
@@ -0,0 +1,224 @@
+        "You have restock alerts enabled for": "Vous avez activé les alertes de réapprovisionnement pour",
+        "You must be an administrator to set channel restock alerts.": "Vous devez être un administrateur pour définir les alertes de réapprovisionnement des canaux.",
+        "You must supply your [TarkovTracker API token](https://tarkovtracker.io/settings/) to link your account.": "Vous devez fournir votre [jeton API TarkovTracker] (https://tarkovtracker.io/settings/) pour lier votre compte.",
+        "You need to specify a search term": "Vous devez spéficfier un terme de recherche",
+        "You need to specify an ammo type": "Vous devez spécifier un type de munitions",
+        "Your hideout progress will update from TarkovTracker {{updateTime}}.": "La progression de votre refuge sera mise à jour à partir de TarkovTracker {{updateTime}}."
+    },
+    "command": {
+        "about": "À propos",
+        "about_desc": "Vous donne quelques informations sur le Bot",
+        "alert": "alerte",
+        "all_desc": "Tout",
+        "ammo": "munitions",
+        "ammo_desc": "Obtenir des informations sur les munitions",
+        "ammo_name_desc": "Entrez le type de munitions",
+        "barter": "troquer",
+        "barter_desc": "IDK",
+        "boss": "boss",
+        "boss_desc": "Obtenir des informations détaillées sur un boss",
+        "boss_select_desc": "Choisir un boss",
+        "changes": "changements",
+        "changes_desc": "Obtenez les derniers changements d'EFT",
+        "channel": "canal",
+        "command": "cammande",
+        "craft": "frabriquer",
+        "craft_desc": "Trouvez des recettes avec un objet spécifique",
+        "crafting_skill_desc": "Frabrication",
+        "flea_market_fee": "IDK",
+        "help": "aide",
+        "help_command_desc": "Obtenez de l'aide à propos d'une commande",
+        "help_desc": "Vous donne quelques informations sur les commandes du Bot",
+        "hideout": "refuge",
+        "hideout_management_level": "IDK",
+        "hideout_management_skill_desc": "Gestion du refuge",
+        "intel_center_level": "IDK",
+        "invite": "inviter",
+        "invite_desc": "Obtenez un lien d'invitation pour inviter le Bot à un autre serveur Discord",
+        "issue": "Problèmes",
+        "issue_desc": "Envoyez vos problèmes aux développeurs",
+        "issue_message_desc": "Écrivez votre message",
+        "level": "niveau",
+        "link": "lien",
+        "map": "carte",
+        "map_desc": "Obtenez des informations détaillées sur une carte",
+        "map_select_desc": "Sélectionnez une carte",
+        "message": "message",
+        "name": "nom",
+        "name_search_desc": "Nom de l'objet à rechercher",
+        "patchnotes": "notes de patch",
+        "patchnotes_desc": "Obtenez les dernières notes de patch",
+        "price": "prix",
+        "price_desc": "IDk",
+        "progress": "IDK",
```

Comment:
I think the literal translation for "progress" in FR would be fine here

Reply:
I was hesitating between "progrès" (when the action stopped and the thing has progressed) and "progression" (the action is still going on like a loading bar/screen). I don't know if this difference exist in English and I couldn't find any info in a dictionary or in my lessons.